app/logic/Calendar/Invitation.ts

```diff
@@ -1,20 +1,24 @@
 export type iCalMethod = "CANCEL" | "REQUEST" | "REPLY";
 
 /**
- * For an inbox item that represents a scheduling message, the type of message:
- * Accepted/Tentative/Declined responses, invitations, or cancellations.
+ * For an inbox item that represents a scheduling message, the type of message.
  */
-export enum Scheduling {
+export enum InvitationMessage {
   None = 0,
-  Accepted = 1,
-  Tentative = 2,
-  Declined = 3,
-  Request = 4,
-  Cancellation = 5,
+  Invitation = 4, // METHOD:REQUEST, an incoming invitation
+  CancelledEvent = 5, // METHOD:CANCEL, a follow-up cancellation
+  ParticipantReply = 6, // METHOD:REPLY, a reply to your outgoing invitation
 }
 
-/* Note: These are EWS/OWA names and ActiveSync values. */
-export enum ResponseType {
+/**
+ * For meetings only, these values indicate the status of either:
+ * - For any given participant, their participation in an event
+ * - The user's own participation in an event
+ *   In particular, this identifies whether the user organised this meeting.
+ *
+ * Note: These are EWS/OWA names and ActiveSync values.
+ */
+export enum InvitationResponse {
   Unknown = 0,
   Organizer = 1,
   Tentative = 2,
```

Comment:
```suggestion
  /** The participant has notified us that he is unsure whether he will attend or not.
   For incoming invitations only. */
  Tentative = 2,
```

Reply:
As a status on an event, this shows your latest response back to the organiser. (Your participant object will also have that status.)

For meetings you organise, this is set on attendees to show that you have seen a response from that attendee (and what it is).